internal/cel/library/sigstore.go

```diff
@@ -0,0 +1,682 @@
+// The first argument is the issuer and the second argument is the subject.
+//
+//	<VerifierBuilder>.keyless(<string>, <string>) <KeylessVerifier>
+//	<KeylessVerifier>.keyless(<string>, <string>) <KeylessVerifier>
+//
+// Examples:
+//
+//	kw.sigstore.image('image:latest').keyless('issuer1', 'subject1').keyless('issuer2', 'subject2') // returns a verifier that verifies the signature of the 'image:latest' image using keyless signing with the keyless info 'issuer1'='subject1' and 'issuer2'='subject2'
+//
+// keylessPrefix
+//
+// Builds a verifier that verifies the signature of an image using keyless signing.
+// The first argument is the issuer and the second argument is a subject as an URL prefix.
+// The provided subject is sanitized to ensure it is a valid URL prefix and to prevent typosquatting.
+// The signature is satisfied only if the subject is a prefix of the signature subject.
+//
+//	<VerifierBuilder>.keylessPrefix(<string>, <string>) <KeylessPrefixVerifier>
+//	<KeylessPrefixVerifier>.keylessPrefix(<string>, <string>) <KeylessPrefixVerifier>
+//
+// Examples:
+//
+//	kw.sigstore.image('image:latest').keylessPrefix('issuer1', 'https://example.com/').keylessPrefix('issuer2', 'https://example.org/') // returns a verifier that verifies the signature of the 'image:latest' image using keyless signing with the keyless prefix info 'issuer1'='https://example.com/' and 'issuer2'='https://example.org/'
+//
+// github
+//
+// Builds a verifier that verifies sigstore signatures of an image using keyless signatures made via Github Actions.
+// The first argument is the owner and the second argument is the repo.
+//
+//	<VerifierBuilder>.github(<string>, <string>) <GitHubVerifier>
+//
+// Examples:
+//
+//	kw.sigstore.image('image:latest').github('owner1', 'repo1') // returns a verifier that verifies sigstore signatures of the 'image:latest' image using keyless signatures made via Github Actions with the owner 'owner1' and the repo 'repo1'
+//
+// certificate
+//
+// Builds a verifier that verifies sigstore signatures of an image using a user provided certificate.
+//
+//	<VerifierBuilder>.certificate(<string>) <CertificateVerifier>
+//
+// Examples:
+//
+//	kw.sigstore.image('image:latest').certificate('certificate') // returns a verifier that verifies sigstore signatures of the 'image:latest' image using the provided certificate
+//
+// certificateChain
+//
+// Adds a certificate to the certificate verifier's chain.
+//
+//	<CertificateVerifier>.certificateChain(<string>) <CertificateVerifier>
+//
+// Examples:
+//
+//	kw.sigstore.image('image:latest').certificate('certificate').certificateChain('certificate1').certificateChain('certificate2') // returns a verifier that verifies sigstore signatures of the 'image:latest' image using the provided certificate and the certificate chain 'certificate1' and 'certificate2'
+//
+// requireRekorBundle
+//
+// Sets whether the certificate verifier requires a Rekor bundle to be present in the signature.
```

Comment:
I would also explain why this can be useful:

> Having a Rekor bundle allows further checks to be performed, e.g. ensuring the signature has been produced during the validity time frame of the cert. Recommended to set to `true`